### DISCUSSION TITLE
Loading Dialog. It seems that the timeline doesnt exist. Maybe the name is wrong?

### DISCUSSION THREAD

**Discussion by Daesina** [dialogic-godot/dialogic#1917] [Q&A] [answered]
I am just following [the youtube video getting started with dialogic](https://youtu.be/sYjgDIgD7AY?si=D0gHaCbfqsIwM-Ea)
and I didnt do anything particular, just customize the theme, add a text event and drag it to the scene. But it reports some error.
Thank you for help!

![屏幕截图 2023-11-19 201420](https://github.com/coppolaemilio/dialogic/assets/134996892/ea58919c-add5-46a7-bedb-16e8cee8eeac)
![屏幕截图 2023-11-19 201428](https://github.com/coppolaemilio/dialogic/assets/134996892/260bd1aa-573e-4088-8e20-218e4037120e)
![屏幕截图 2023-11-19 201447](https://github.com/coppolaemilio/dialogic/assets/134996892/8030f78a-200b-4c1c-b236-d17fa89f6351)

**Comment by Jowan-Spooner** [comment, marked as answer]
You have to tell the dialogic node which timeline to start (currently none, so it says it cannot find anything). As far as I remember it is the little "timeline" dropdown in the inspector (when the dialogic node is selected):
![grafik](https://github.com/coppolaemilio/dialogic/assets/42868150/f3f26910-5027-4ed8-877b-d6f2599d914b)

**Reply by Daesina** [reply]
Thank you 👍
